governance/polls/MIP9 Community Greenlight Poll - Gelato UniswapV3 DAI-USDC LP Token - G-UNIv3-DAIUSDC - July 19, 2021.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Community Greenlight Poll - Gelato UniswapV3 DAI-USDC LP Token - G-UNIv3-DAIUSDC - July 19, 2021
```

Comment:
Asset code is supposed to be in brackets.

Reply:
In fact these are the wrong way around as well. Template is $asset_code ($asset_name)